src/actions/index.js

```diff
@@ -29,7 +29,12 @@ export const getSearchResultAndPros = query => dispatch => Promise.all([
   dispatch(setSearchQuery(query)),
   dispatch(getSearchResult(query)),
   dispatch(getProPlayers()),
-  dispatch(getMatch(query)),
+  (() => {
```

Comment:
couldn't this just be wrapped in an if statement instead of adding the IIFE?  Maybe turn this into a full function, build the array and then Promise.all that